e2e/LongHaul/service/Program.cs

```diff
@@ -38,6 +38,12 @@ private static async Task Main(string[] args)
                     Environment.Exit(1);
                 });
 
+            if (!parameters.Validate())
+            {
+                Console.WriteLine(CommandLine.Text.HelpText.AutoBuild(result, null, null));
```

Comment:
Console.WriteLine?

Reply:
Yes, this is to the end user who runs the app.